source/_lovelace/map.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: "Map Card"
+sidebar_label: Map
+description: "A card that allows you to display entities on a map"
+date: 2018-07-08 10:28 +00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+---
+
+A card that allows you to display entities on a map
+
+<p class='img'>
+<img src='/images/lovelace/lovelace_map_card.png' alt='Screenshot of the map card'>
+Screenshot of the map card.
+</p>
+
+**Options**
+
+| Name | Type | Default | Description
+| ---- | ---- | ------- | -----------
+| type | string | **Required** | `map`
+| entities | list | **Required** | A list of entities to display on the map. Can be entity_ids or entity objects (see structure below).
+| title | string | Optional | Card title
+| aspect_ratio | string | `"100%"` | Map height:width ratio
+
```

Comment:
{% configuration %}
type:
  required: true
  description: map
  type: string
cards:
  required: true
  description: A list of entities to display on the map. Can be entity_ids or entity objects (see structure below).
  type: list
title:
  required: false
  description: Card title
  type: string
aspect_ratio:
  required: false
  description: Map height:width ratio
  type: string
{% endconfiguration %}